server/spinwick.go

```diff
@@ -655,12 +671,14 @@ func checkMMPing(client *mattermostModel.Client4, ctx context.Context) error {
 	}
 }
 
-func makePullRequestID(repoName string, prNumber int) string {
+func makeSpinWickID(repoName string, prNumber int) string {
 	return strings.ToLower(fmt.Sprintf("%s-pr-%d", repoName, prNumber))
 }
 
-func NewBool(b bool) *bool       { return &b }
-func NewInt(n int) *int          { return &n }
-func NewInt64(n int64) *int64    { return &n }
-func NewInt32(n int32) *int32    { return &n }
-func NewString(s string) *string { return &s }
+func isSpinWickLabel(label string) bool {
+	if label == Config.SetupSpinWick || label == Config.SetupSpinWickHA {
+		return true
+	}
+
+	return false
```

Comment:
nit: could be simplified to `return Config.SetupSpinWick || label == Config.SetupSpinWickHA`